Formula/arangodb.rb

```diff
@@ -41,6 +45,13 @@ class Arangodb < Formula
     EOS
   end
 
+  # https://github.com/arangodb/arangodb/issues/17454
+  # https://github.com/arangodb/arangodb/issues/17454
+  fails_with :gcc do
+    version "11"
```

Comment:
This will reject `gcc <= 11`. With current DSL, will want to use `fails_with gcc: "11"` which rejects `gcc == 11`.